helm/efk-stack-app/charts/opendistro-es/templates/elasticsearch/es-client-ingress.yaml

```diff
@@ -42,8 +46,11 @@ spec:
       http:
```

Comment:
Also needs  the`ingressClassName` field. I suggest adding a new value into the chart values so that the user is able to change it.

Reply:
Please review